src/js/control-bar/skip-buttons/skip-backward.js

```diff
@@ -0,0 +1,69 @@
+import Button from '../../button';
+import Component from '../../component';
+
+/**
+ * Button to skip backward a configurable amount of time
+ * through a video. Renders in the control bar.
+ *
+ *  * e.g. options: {controlBar: {skipButtons: backward: 5}}
+ *
+ * @extends Button
+ */
+class SkipBackward extends Button {
+  constructor(player, options) {
+    super(player, options);
+
+    this.validOptions = [5, 10, 30];
+    this.skipTime = this.getSkipBackwardTime();
+
+    if (this.skipTime && this.validOptions.includes(this.skipTime)) {
+      this.controlText(`Skip backward ${this.skipTime} seconds`);
```

Comment:
This way means the translation file need a translation for each allowed increment. It's relatively manageable with 3 permitted values, but it is [possible to use a variable](https://github.com/videojs/video.js/blob/e1a7ac376a2b0cb69f92bff45f6ad2b0fbc1a55d/src/js/loading-spinner.js#L25) in these translations: `this.localize('Skip backward {seconds} seconds', [this.skipTime])`

Either way there should be at least an entry for each string in en.json, so there's a template to translate from.

Reply:
I agree.

It should be safe to pass a pre-localized string to `controlText`, like:
```js
this.controlText(this.localize('Skip backward {seconds} seconds', [this.skipTime]));
```